hass/devices.js

```diff
@@ -214,6 +214,27 @@ module.exports = {
       }
     }
   ],
+  '411-515-3': [
+    // Heatit Thermostat TF 058 (ThermoFloor AS)
+    {
+      type: 'climate',
+      object_id: 'thermostat',
```

Comment:
@hwestman Isn't it missing the `"setpoint_topic": { "Heat (Default)": "67-1-1", "Cool": "67-1-2" },`?

Reply:
"setpoint_topic: Key-Value object where keys are the modes of the Zwave thermostat and values are the matching setpoint value_id (use this if your thermostat has more than one setpoint)"

My conclusion was that it didn't have more than one setpoint, so i removed it. I'll look into it!

Reply:
Sorry just noticed that you only have heat mode :) It's ok so